hugo.yaml

```diff
@@ -48,6 +48,10 @@ params:
             # 👇 and enter it here.
             sessionizeId:
             title: Your Event Title
+            # 👇 An optional short version of your event's title. Only use it if
+            # 👇 your event title is very long. The short title may be used in
+            # 👇 places where space is limited (e.g., in the header menu).
+            shortTitle:
```

Comment:
I removed the default value here, because the configuration files get merged. So if a website is not using the short title, the very long default short title was used (resulting in the header menu not being displayed).